File: Pizza_Calculator.cpp
#include <iostream>
#include <iomanip>
using namespace std;
// getting volume and dimentions
int main() {

int people, pizza, slices, slicesLeft, totalSlices, slicesPerperson;

cout << "Amount of people: " << endl;
cin >> people;
cout << "Amount of boxes of pizzas: " << endl;
cin >> pizza;
cout << "Slices Per Pizza: "<< endl;
cin >> slices;

totalSlices = pizza * slices;

slicesPerperson = 3 * people;
slicesLeft = totalSlices - slicesPerperson;

cout << "Slices left over: " << slicesLeft << endl;

    return 0;
}

File: Pizza_Calculator.js
let people, pizza, slices, totalSlices, slicesPerperson, slicesLeft;

console.log("Amount of people: ");
people = parseInt(prompt());

console.log("Amount of boxes of pizzas: ");
pizza = parseInt(prompt());

console.log("Slices Per Pizza: ");
slices = parseInt(prompt());

totalSlices = pizza * slices;

slicesPerperson = 3 * people;
slicesLeft = totalSlices - slicesPerperson;

console.log("Slices left over: " + slicesLeft);